packages/replicache/src/kv/sqlite-store.bench.ts

```diff
@@ -0,0 +1,168 @@
+import {afterAll, bench, describe, expect} from 'vitest';
+import {withRead, withWrite} from '../with-transactions.ts';
+import {getTestSQLiteDatabaseManager} from './sqlite-store-test-util.ts';
+import {SQLiteStore} from './sqlite-store.ts';
+
+const walSQLite3DatabaseManager = getTestSQLiteDatabaseManager();
+const walStore = new SQLiteStore('bench-wal', walSQLite3DatabaseManager, {
+  journalMode: 'WAL',
+});
+
+const defaultSQLite3DatabaseManager = getTestSQLiteDatabaseManager();
+const defaultStore = new SQLiteStore(
+  'bench-default',
+  defaultSQLite3DatabaseManager,
+);
+
+afterAll(() => {
+  walSQLite3DatabaseManager.clearAllStoresForTesting();
+  defaultSQLite3DatabaseManager.clearAllStoresForTesting();
+});
+
+describe('zero-sqlite3 WAL', () => {
+  bench(
+    `write`,
+    async () => {
+      await withWrite(walStore, async wt => {
+        await wt.put('foo1', 'bar1');
```

Comment:
what is the bench attempting to measure? Just raw _transactions_ per second? Or do you want more information on _writes_ per second?

Writes per second should be roughly on the order of `transactions_per_second * writes_per_transaction` so 1 write per transaction will give you the same writes per second as transactions per second.

Reply:
Yeah this is a good question. I was using this benchmark to improve performance of the SQLite store w/ better-sqlite3, as a way to test performance of the store as I was making changes to schema/PRAGMAs. I updated this to be simpler and compare WAL modes. Let me know if this doesn't fit into existing benchmarks or any ways you see improving this.